front/lib/api/assistant/conversation.ts

```diff
@@ -223,16 +221,9 @@ export async function getUserConversations(
     attributes: ["id", "sId", "title", "createdAt"],
     include: [
       {
-        model: Message,
+        model: ConversationParticipant,
+        where: { userId: user.id, action: "posted" },
```

Comment:
Shouldn't this be done the other way around? 

If not, is this correclty optimized?